### ISSUE TITLE
Please either close or accept open pull requests.

### ISSUE THREAD

**Issue by leycec** [gentoo/sci#918]
There are [currently 35 open pull requests](https://github.com/gentoo/sci/pulls) as of this writing – 7 of which have yet to receive a single comment. That's... *not the best.*

Delays in resolving volunteer contributions to a third-party overlay are, of course, not unexpected. What is unexpected is how many requests have been left open seemingly indefinitely. That's... *not the best.*

**We can do better.** Ideally:

* Pull requests older than two or three years (*...yes, there are many of these*) may need to simply be closed with sincere apologies. Most of these requests are likely to suffer non-trivial conflicts that have accrued over the years. The minority of requests that suffer no such conflicts are unlikely to build as expected against a modern Portage tree. In short, there's not much that can be profitably done here. It's time to say goodbye to rotten fruit. :apple: 
* For newer pull requests, new comments should be posted to these requests by overlay administrators politely requesting that all pending issues be resolved. If there *are* no pending issues, the request should either be rejected out of hand (*for... reasons*) or accepted as is.

In summary, to quote [an esteemed poet](https://www.youtube.com/watch?v=e7qQ6_RV4VQ):

> Come gather 'round, people
Wherever you roam
And admit that the waters
Around you have grown
And accept it that soon
You'll be drenched to the bone
If your time to you is worth savin'
And you better start swimmin'
Or you'll sink like a stone
For the times they are a-changin'

**Comment by heroxbd** [comment]
Thank you Cecil for reminding us!

**Comment by leycec** [comment]
Thank *you*! Everyone obviously has better things to do with scarce summertime than close obsolete PRs. Nonetheless, out of courtesy to external contributors who generously volunteered for the Good of Gentoo Kind™, it'd be great if we at least acknowledged their efforts and apologized a bit for failing to resolve them in a timelier manner.

**Anyway!** We now return to your regularly scheduled Gentoo-based scientific research efforts. :books:

**Comment by TheChymera** [comment]
@leycec we are currently working on this, thank you. The issue was that of the active developers, none had maintainer access to the GitHub repo specifically.
